docs/medical-api/more-info/pagination.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: "Pagination"
+description: "Retrieving larger amounts of data."
+---
+
+Some of Metriport's endpoints are paginated: they return a limited amount of items each time they are requested. 
+Each request being a single "page".
+
+In order to have access to all items, after we receive the items for the current page we can request the
+next page to the API.
+
+Each response includes:
+- `meta`: the information about the response.
+- `<items>`: the list of items for the respective request, where `items` is named according to the respective
+  type of data being requested - e.g., `patients`, `documents`.
+
+And `meta` contains:
+- `itemsOnPage`: the amount of items on the current page.
+- `nextPage`: (optional) the URL to return the next page; if not present it means there's not a next page
+  and all items have been returned.
+- `prevPage`: (optional) the URL to return the previous page; if not present it means we're on the first page.
+
+## Pagination parameters
+
+Metriport's pagination is designed simplify its usage, hiding away the complexity for most situations.
+
+But it can be customized using these parameters:
+- `count`: the number of items to be included in each page/response, optional. Defaults to 50 items per page.
+  The maximum number of items per page is 500.
+- `fromItem`: the ID of the first item to be included in the current page, optional. If not provided, the
+  first page will be returned.
+- `toItem`: the last item to be included in the current page, optional. If not provided, it will be
+  automatically be calculated based on `firstItem` and `count`.
+
+Only two of the parameters above can be specified for each request.
+
+## Pagination in the SDK
+
+The SDK includes support for the paginated endpoints in two ways:
+- a simpler "page" version of the function, in which we can just pass the respective page's URL;
+- a more detailed version of the function, in which we can specify each pagination parameter individually.
```

Comment:
Could we show this example as well

Reply:
It's the last example on the page. I updated the text before each example to try and make it clear the relationship to each item on this list.